### ISSUE TITLE
Can't run on windows 10: "Not allowed to load local resource"

### ISSUE THREAD

**Issue by CS-Jfilmer** [zeebe-io/zeebe-modeler#73]
Steps
- Download zeebe modeler 0.6.2
- Unpack
- Attempt to run Zeebe Modeler.exe

![2019-03-25 08_50_14-index js - zeebe-modeler - Visual Studio Code](https://user-images.githubusercontent.com/48894163/54887143-3a094080-4edb-11e9-8bc8-fb78ba83fdff.png)

- Tried running as admin

Environment
 - OS: Windows 10
 - Zeeber Modeler Version: [e.g. 0.6.2-win-64]

---

Depends on https://github.com/zeebe-io/zeebe-modeler/issues/79

**Comment by CS-Jfilmer** [comment]
Figured it out, if you have NODE_ENV as "development" then running the exe will fail. Makes using the modeler on a development machine a little tricky

**Comment by nikku** [comment]
You may unset the environment variable before running the app.

Works like this on Linux, not sure about Windows though:

```
NODE_ENV= camunda-modeler
```

**Comment by nikku** [comment]
Generally speaking you should probably avoid making NODE_ENV a global variable.

On windows for example `cross-env` can help you to inject the variable only when required:

```
cross-env NODE_ENV=production node server.js
```

**Comment by nikku** [comment]
Closed in favor of https://github.com/camunda/camunda-modeler/issues/1352.

**Comment by nikku** [comment]
Fixed upstream in (`camunda-modeler@3.1.0`](https://github.com/camunda/camunda-modeler/blob/master/CHANGELOG.md#310).
